real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/sensors/CameraNode.java

```diff
@@ -86,10 +90,19 @@ public void actionPerformed(String topicName, Message m) {
 					if (status.equals(LoggingNode.LoggingNodeStatus.STARTED_LOGGING)) {
 						JsonObject params = message.getParams();
 						String outputDir = params.get("outputDir").getAsString();
+
+						if (!webcam.isOpen()) {
+							webcam.open();
+						}
 						videoEncoder = new SequenceEncoder(new File(outputDir + "/webcam.mp4"));
```

Comment:
not really, even with this change, it doesn't help. #416 is mainly addressing between runs, where we fail to release the lock on shutdown